corehq/warehouse/transforms/sql/app_status_fact.sql

```diff
@@ -0,0 +1,45 @@
+SELECT * FROM
```

Comment:
this is a fairly complex sql query that i still need to tweak a bit. for one i need to ensure its accuracy and two need to join it with the user dim. i'll also need to document this sql query to make it easier to understand, but these are the types of queries we're looking at...

this currently returns:
```
{'build_id': None, 'user_id': u'a191f7eba69a44c9bf80e0f8ec4c7923', 'app_id': u'be5192c04e0ff690cd6de44fdae9531b', 'last_sync': datetime.datetime(2017, 2, 1, 10, 11, 4, 182407), 'last_submission': datetime.datetime(2017, 2, 1, 14, 13, 51, 283805), 'last_submission_id': u'399929e1-458c-4be0-a933-64570c03864e'}
```